module/season_to_team_stats.rb

```diff
@@ -0,0 +1,58 @@
+module SeasonToTeamStats
+
+  def seasonal_info(seasonid)
```

Comment:
This helper method seasonal_info sets up our data so we can calculate the rest of iteration 5 information